.vscode/launch.json

```diff
@@ -71,6 +71,20 @@
       "showDevDebugOutput": "none",
       "preLaunchTask": "Build: DIM.elf"
     },
+    {
+      "name": "Debug Embedded: h7dev",
```

Comment:
Good for now, may need to re-think how to clearly list which boards are h7 vs F4 in the future